.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## RVM Version
```

Comment:
I think we could make it a bit more structural and in the order which is easier to read:

## Steps to reproduce:
*
*
*

## Expected behavior

## Actual behavior

## Environment
* Operating system: `Windows 7, macOS 10.11 El Capitan, Ubuntu 14.0.4`
* Shell: `bash, sh, zsh, cygwin`
* RVM version: `rvm --version`